### DISCUSSION TITLE
Microfrontend in Quasar

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#14333] [General - Components / Directives / etc]
Dear Team , 

We are developing huge Enterprise application with Quasar , we want to implement Micro frontend concept , because we want to develop independent modules separately so that deployment & testing would be easy.
Please guide us with this.

**Comment by TobyMosque** [comment]
there is some options, but most of them will not work well with the Quasar CLI.

Vite/Vue CLI:
**Microfront Framework**: https://single-spa.js.org/docs/ecosystem-vue#vue-3
**Web Components**: https://vuejs.org/guide/extras/web-components.html#building-custom-elements-with-vue

Quasar CLI:
**UI Extensions**: https://quasar.dev/app-extensions/tips-and-tricks/starter-kit-equivalent

in the case of `Web Components`, `UI Extensions`, i recommend you to use something `Yarn Workspaces` and `Git Submodules`, where each microfrontend will lie in your own workspace/repository.
https://blog.nrwl.io/dev-workflow-using-git-submodules-and-yarn-workspaces-14fd06c07964

**Reply by jmriyaz84** [reply]
Thanks @TobyMosque for your quick response.
